config/scenario_config_21_EU11_ECEMF.csv

```diff
@@ -47,7 +47,7 @@ xx_WP1_NDC_debug;0;2030.EUR_regi.tax.t.FE_wo_b_wo_n_e.all 1.2809;;;;;;;;;;;;;reg
 EWN_DIAG-C400-lin-ResidualFossil_debug;0;2030.EUR_regi.tax.t.FE_wo_b_wo_n_e.all 1.1916;on;;highFossilPrice;;;;;;;;;;regiCarbonPrice;;on;on;EUR_regi, NEU_regi;GLO.(2025 116,2030 196,2035 277,2040 357,2045 438,2050 518,2055 598,2060 679,2070 839,2080 1000,2090 1161,2100 1321,2110 1321,2130 1321,2150 1321);testOneRegi;EWN;debug;1;;;Mix4;;;./config/regionmapping_21_EU11.csv;./config/extramapping_EU27.csv;rcp20;;globallyOptimal;0;;;;9;;NDC;;heat;2025;xx_DIAG-C400-lin;xx_DIAG-C400-lin;xx_DIAG-Base;;
 # ________REMIND_3p0p0_cplx________;0;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 # _____Calibration_cplx_____;0;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-xx_cplx_Calib_FE_med;0;;;;;;;;;;apcardiEfft 0.8, apcardiEffH2t 0.675;;FE_med;medium;;;;;EUR_regi, NEU_regi;;;;;;;complex;Mix1;fixed_shares;calibrate;./config/regionmapping_21_EU11.csv;./config/extramapping_EU27.csv;;;;0;;;;;;none;;heat;2005;;;;;
+#  xx_cplx_Calib_FE_med(needs adjustment regarding cm_calibration_FE);0;;;;;;;;;;apcardiEfft 0.8, apcardiEffH2t 0.675;;FE_med;medium;;;;;EUR_regi, NEU_regi;;;;;;;complex;Mix1;fixed_shares;calibrate;./config/regionmapping_21_EU11.csv;./config/extramapping_EU27.csv;;;;0;;;;;;none;;heat;2005;;;;;
```

Comment:
Scenarios that use transport complex can be removed from this scenario config, removing all scenarios dependable of the deprecated `cm_calibration_FE` from the scenario config as consequence.
This can be included in this PR or I could add this change afterwards. Whatever you prefer.

Reply:
I think, the whole config file needs a review, so please do it afterwards